src/lib/edge-functions/proxy.ts

```diff
@@ -136,11 +137,9 @@ export const initializeProxy = async ({
 }) => {
   const userFunctionsPath = config.build.edge_functions
   const isolatePort = await getAvailablePort()
-  const buildFeatureFlags = {
-    edge_functions_npm_modules: true,
-  }
   const runtimeFeatureFlags = ['edge_functions_bootstrap_failure_mode', 'edge_functions_bootstrap_populate_environment']
   const protocol = settings.https ? 'https' : 'http'
+  const buildFeatureFlags = { ...getFeatureFlagsFromSiteInfo(siteInfo), edge_functions_npm_modules: true }
```

Comment:
This flag is not used anymore. We'll be able to get rid of this special case once we update `@netlify/edge-bundler`.